### ISSUE TITLE
Error? Not sure why this is happening

### ISSUE THREAD

**Issue by IsaiahT-Tech** [IsaiahT-Tech/cryptography-api#14]
![image](https://user-images.githubusercontent.com/63564966/90211189-e5eb4e80-dda4-11ea-9ef6-3ca7870d893e.png)
`routers/crypto.py`

#### PLEASE READ (there will be many mistakes of course, though I can't import models into, as it says it doesn't exist. Now we may have different ideas, but here is mine and what I am trying to do

* Our ORM is `tortoise-orm`
* Using `tortoise-orm` to get store the text input inside
* Models can't be found as shown by the screenshot above

Here is another one of `models/models.py`

![image](https://user-images.githubusercontent.com/63564966/90211386-5d20e280-dda5-11ea-85ca-62d61e30febc.png)

#### Now of course there will be mistakes of course. It would be nice for a simple explanation of this. (Still under development) 
-> it would be nice for this to be answered by 12:00 PM if not then we will have to extend this task to Sunday or during conference chat for this project. 

*If you have different ideas if what we are doing we should scrap this project as we may not be communicating correctly. We may have to go back to design board to make sure we are 100% on par on ideas*

If the topic of this is not what you have planned to do we will STOP development of this project and go back to the drawing board. There are some development and planning issues in our small team. ( It would be great to get more to actually map out this project )

#### If everything I said does not apply, please answer ASAP :smile: :sweat_smile: :man_technologist: :disappointed: :smiley:

**Comment by SnkSynthesis** [comment]
@IsaiahT-Tech 

Nice work so far!

I think the first API should be for giving text that can be encrypted and then returning a `pydantic` model which contains the key and the token and then you can use the key and the token to decrypt it later. Once we get this done we can make another API that can allow us to store encrypted values and decrypt when want (requires a frontend (`jinja2` is good) and `tortoise-orm` so complexity is much higher so that is why I am saying we should do this first as it does the implementation of what we want but in a simple way but once this is done then we can start implementing a more complex method like I have described above)

e.g. (**NOTE**: You may have to change this based on your needs)

```python
class Encrypted(BaseModel):
    token: str
    key: str
```

So basically we can do a `GET` and return a `pydantic` model that stores the key and the encrypted token.
e.g.

```python
@router.get('/{text}')
async def encrypt_text(text: str):
    # Do encrypting etc...
    encrypted = Encrypted(token=token, key=key)  # Encrypted token and generated key
    return encrypted
```

and then we can do a `POST` that takes in a `Encrypted` and decrypts it.

```python
@router.post('/')
async def decrypt_text(encrypted: Encrypted):
    # Do decrypting, etc with the encrypted.key and encrypted.token
    return text
```
